build.psm1

```diff
@@ -615,10 +663,40 @@ function Start-PSPester {
     else {
         try {
             $originalModulePath = $env:PSMODULEPATH
-            
-            & $powershell -noprofile -c $Command
+            if ($Unelevate)
+            {
+                Start-UnelevatedProcess -process $powershell -arguments @('-noprofile', '-c', $Command)
+                $currentLines = 0
+                while ($true)
+                {
+                    $lines = Get-Content $outputBufferFilePath | Select-Object -Skip $currentLines
+                    $lines | Write-Host
+                    if ($lines | ? { $_ -eq '__UNELEVATED_TESTS_THE_END__'})
+                    {
+                        break
```

Comment:
possibly, i believe that appveyor has it's own time limitations on execution, it may be that nothing is needed here (or rather, any number you select will probably be wrong, up to the value that AppVeyor has)